wp-content/themes/core/components/blocks/interstitial/Interstitial_Block_Controller.php

```diff
@@ -0,0 +1,202 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Templates\Components\blocks\interstitial;
+
+use Tribe\Libs\Utils\Markup_Utils;
+use Tribe\Project\Blocks\Types\Hero\Model;
+use Tribe\Project\Blocks\Types\Interstitial\Controller;
+use Tribe\Project\Blocks\Types\Interstitial\Interstitial as Interstitial_Block;
+use Tribe\Project\Templates\Components\Abstract_Controller;
+use Tribe\Project\Templates\Models\Image;
+use Tribe\Project\Theme\Config\Image_Sizes;
+
+/**
+ * Class Interstitial
+ */
+class Interstitial_Block_Controller extends Abstract_Controller {
+
+	public const LAYOUT            = 'layout';
+	public const MEDIA             = 'media';
+	public const CONTENT           = 'content';
+	public const CTA               = 'cta';
+	public const CONTAINER_CLASSES = 'container_classes';
+	public const MEDIA_CLASSES     = 'media_classes';
+	public const CONTENT_CLASSES   = 'content_classes';
+	public const CLASSES           = 'classes';
+	public const ATTRS             = 'attrs';
+
+	public string $layout;
+	public ?int $media;
```

Comment:
typo?

Reply:
nullable

Reply:
In the instance that an image isn't provided, ACF will return an empty string. If it's provided, an int. So I'm setting default to null and allowing this property to be nullable or an int. the `?` is the way to set it as nullable.

Reply:
Learn something new every day 😃